android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationOptions.kt

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.translate
+
+/**
+ * Translation options that map to the Gecko Translations Options.
+ *
+ * @property downloadModel If the necessary models should be downloaded on request. The translation will
+ * complete exceptionally if the models are not already available.
```

Comment:
```suggestion
 * @property downloadModel If the necessary models should be downloaded on request. The translation will
 * complete except if the models are not already available.
```
❓I believe this is what you intended but I could be wrong.

Reply:
Thanks, adjusted language